### DISCUSSION TITLE
How would I group updates to Github maintained actions together?

### DISCUSSION THREAD

**Discussion by benedfit** [renovatebot/renovate#14899] [Request Help] [answered]
I currently have two separate PR:

- chore(deps): update actions/cache action to v3
- chore(deps): update actions/setup-node action to v3

If I was to add the group below, would this combine the two, or do Github Actions need to be grouped differently?

```
packageRules: [
  {
    "description": "All Github maintained actions"
    "groupName": "Github maintained actions",
    "groupSlug": "github-actions",
    "matchSourceUrlPrefixes": "https://github.com/actions"    
  }
]
```

Thanks in advance

**Comment by rarkins** [comment, marked as answer]
I think you'd want to use `matchManagers` and `matchPackagePrefixes`
